### PR TITLE
Networked VM e2e tests

### PR DESCRIPTION
**Pull request by jakub-dzon**
```release-note
NONE
```
Signed-off-by: Jakub Dzon <jdzon@redhat.com>

### CODE REVIEW

File: cluster-sync/imageio/imageio.yaml [hunk 1]
```diff
@@ -22,7 +22,8 @@ spec:
         ports:
         - containerPort: 12345
       - name: fakeovirt
-        image: machacekondra/fakeovirt:test
+        # update back to original image when changes are merged
```
Comment (pkliczewski) [inline cluster-sync/imageio/imageio.yaml]: How do we track this change?

File: cluster-sync/imageio/imageio.yaml [hunk 2]
```diff
@@ -22,7 +22,8 @@ spec:
...
+        image: quay.io/jdzon/fakeovirt:latest
```
Comment (machacekondra) [inline cluster-sync/imageio/imageio.yaml]: I've updated the repo with your changes, thanks.

File: cluster-sync/imageio/imageio.yaml (hunk 1 above)
Comment (jakub-dzon) [reply cluster-sync/imageio/imageio.yaml]: Ondra has pushed the new image - I will update it in a moment.

File: cluster-sync/imageio/imageio.yaml (hunk 2 above)
Comment (jakub-dzon) [reply cluster-sync/imageio/imageio.yaml]: Thanks!

File: cluster-sync/imageio/imageio.yaml (hunk 1 above)
Comment (jakub-dzon) [reply cluster-sync/imageio/imageio.yaml]: Done.

File: tests/basic_net_vm_import_test.go [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+var _ = Describe("Networked VM import ", func() {
+	var (
+		f         = framework.NewFrameworkOrDie("networked-vm-import")
+		secret    corev1.Secret
+		namespace string
+		test      = networkedVmImportTest{}
+	)
+
+	BeforeEach(func() {
+		namespace = f.Namespace.Name
```
Comment (masayag) [inline tests/basic_net_vm_import_test.go:31]: Is the namespace on which the operator is deployed is used also for creating the vm-import resources and the VM ?
If this is the case, perhaps we'd like to make sure they are different, so there aren't any assumptions regarding permissions ?

Comment (jakub-dzon) [reply tests/basic_net_vm_import_test.go:31]: No, it's not. 
`framework.Framework` creates namespace with generated name and prefix given in its constructor:
https://github.com/kubevirt/vm-import-operator/blob/b21b0d4961f4789b51292ea715f1e7186ac96f7e/tests/framework/framework.go#L159

**Review by pkliczewski — APPROVED** [review]

**Review by masayag — APPROVED** [review]

**Review by machacekondra — APPROVED** [review]

**Pull request merged**